bluetooth.go

```diff
@@ -335,6 +336,18 @@ func (d *Device) hci_advance_read_ringbuf(n uint32) error {
 	return err
 }
 
+func (d *Device) hci_ringbuf_debug() (h2btio_bt2hio [4]uint32) {
+	buf8 := u32AsU8(h2btio_bt2hio[:])
+	d.bp_read(d.btaddr+whd.BTSDIO_OFFSET_HOST2BT_IN, buf8)
+	d.trace("hci_ringbuf_debug",
+		slog.Uint64("h2bt_in", uint64(h2btio_bt2hio[0])),
+		slog.Uint64("h2bt_out", uint64(h2btio_bt2hio[1])),
+		slog.Uint64("bt2h_in", uint64(h2btio_bt2hio[2])),
+		slog.Uint64("h2bt_out", uint64(h2btio_bt2hio[3])),
```

Comment:
Did you mean `bt2h_out` here?
```suggestion
		slog.Uint64("bt2h_out", uint64(h2btio_bt2hio[3])),
```